docker/locks/README.md

```
@@ -60,18 +60,29 @@ image-specific dependencies that are intentionally excluded from the shared CPU
 
 ```bash
-uv lock --project docker/locks/nmp-gym-tasks --python 3.13.14
+uv lock --project docker/locks/nmp-gym-tasks --python 3.13.15
 ```
 
 Verify both image architectures:
 
+:::::{tab-set}
+
+::::{tab-item} x86_64
 ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)